react/week1/ToDo1.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const ToDo1 = (props) => {
+  const { style } = props;
+  const { activity, time } = props.task;
+  console.log(style.color);
```

Comment:
Normally you would remove console.log statements from code that is merged into the `master` branch.